tests/test_downloader_handlers.py

```diff
@@ -687,9 +687,6 @@ class BaseFTPTestCase(unittest.TestCase):
     password = "passwd"
     req_meta = {"ftp_user": username, "ftp_password": password}
 
-    if six.PY3:
-        skip = "Twisted missing ftp support for PY3"
```

Comment:
Do you think it still should be skipped for unsupported Twisted versions in Python 3? Or do we support only latest Twisted in Python 3?